addon/helpers/github-issue.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function githubIssue([title]/*, hash*/) {
+	let titleStr = encodeURIComponent(`Demo App Test fail: ${title}`);
+	let linkStr = 
+		`https://github.com/nathanhammond/ember-a11y/issues/new?title=${titleStr}`;
+  return linkStr;
+}
+
+export default Ember.Helper.helper(githubIssue);
```

Comment:
We should move this entire helper into the test application instead of in the top-level addon.